### ISSUE TITLE
travis build jobs fail

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1968]
It seems like the Jenkins build jobs do not check for bashism, but only the travis build jobs do?
(At least I only saw [failing travis jobs](https://travis-ci.org/ElektraInitiative/libelektra/jobs/375953571) but no failing Jenkins jobs)

**Comment by ingwinlu** [comment]
I checked with the console output of the jenkinsfile and I see that bashism
checks get run there. can you pinpoint a pr where travis fails because of
it and jenkins passes?

**Comment by ingwinlu** [comment]
nvm
```
[debian-oldstable-full] ELEKTRA SCRIPTS BASHISMS TEST
[debian-oldstable-full] 
[debian-oldstable-full] checkbashisms command needed for this test, aborting
```

**Comment by ingwinlu** [comment]
this will be resolved with #1965 as I install devscripts into the docker image for debian stretch

**Comment by markus2330** [comment]
Thanks for the quick answer, I updated the docu in b23c4dcc1b24a9daeb34ad288336025f0a63ca40
